akka-actor/src/main/scala/akka/event/Logging.scala

```diff
@@ -1289,13 +1285,17 @@ trait LoggingAdapter {
     case Logging.WarningLevel ⇒ isWarningEnabled
     case Logging.InfoLevel    ⇒ isInfoEnabled
     case Logging.DebugLevel   ⇒ isDebugEnabled
+    case _                    ⇒ false
   }
 
   final def notifyLog(level: Logging.LogLevel, message: String): Unit = level match {
     case Logging.ErrorLevel   ⇒ if (isErrorEnabled) notifyError(message)
     case Logging.WarningLevel ⇒ if (isWarningEnabled) notifyWarning(message)
     case Logging.InfoLevel    ⇒ if (isInfoEnabled) notifyInfo(message)
     case Logging.DebugLevel   ⇒ if (isDebugEnabled) notifyDebug(message)
+    case unsupported ⇒ if (isEnabled(unsupported)) {
```

Comment:
Why is this guarded by the if?

Reply:
Why would we throw an error if logging not enabled

Reply:
Also I believe you requested it, unless I misunderstood.